### PR TITLE
Add support for setting parent project

### PR DESCRIPTION
**Pull request by priteau**
This is not supported by the openstack.cloud.project Ansible module, and not even by its dependency openstacksdk. Use openstackclient instead.

### CODE REVIEW

File: tasks/projects.yml [hunk 1]
```diff
@@ -78,22 +78,50 @@
...
+    {% if item.project_domain is defined %}
+    --domain '{{ item.project_domain }}'
+    {% endif %}
+    {% if item.parent is defined %}
+    --parent '{{ item.parent }}'
+    {% endif %}
   with_items: "{{ os_projects }}"
-  vars:
-    domain_is_id: "{{ item.project_domain in os_projects_domain_to_id.values() }}"
+  when: item.name not in projects_list.stdout_lines
```
Comment (markgoddard) [inline tasks/projects.yml:127]: We would lose the ability to update projects with this change

Comment (priteau) [reply tasks/projects.yml:127]: Good point.

Comment (markgoddard) [reply tasks/projects.yml:127]: Could use a separate `openstack project set` task?

Comment (priteau) [reply tasks/projects.yml:127]: A simpler approach is to keep using `os_project` for updating project information such as description.

Note that it isn't possible to update the parent of a project, I will add a check for this in a follow-up.

**Review by markgoddard — APPROVED** [review]
Nice compomise. A comment would help us to know when it is possible to remove the new code.

**Review by markgoddard — APPROVED** [review]

**Pull request merged**